specs/fr/v1.0.0.md

```diff
@@ -0,0 +1,1052 @@
+![TOML Logo](logos/toml-200.png)
+
+TOML v1.0.0
+===========
+
+Tom's Obvious, Minimal Language.
+
+par Tom Preston-Werner, Pradyun Gedam, etc.
+
+Objectifs
+---------
+
+TOML est un format de fichier de configuration minimal facile à lire grâce à
+une sémantique claire. Il est conçu pour correspondre sans ambiguïté à une
+table de hachage et doit être facile à représenter dans de nombreux langages.
+
+Sommaire
+--------
+
+- [Spécifications](#spécifications)
+- [Commentaire](#commentaire)
+- [Paire clé/valeur](#paire-clévaleur)
+- [Clé](#clé)
+- [Chaîne](#chaîne)
+- [Entier](#entier)
+- [Flottant](#flottant)
+- [Booléen](#booléen)
+- [Date-heure avec offset](#date-heure-avec-offset)
+- [Date-heure locale](#date-heure-locale)
+- [Date locale](#date-locale)
+- [Heure locale](#heure-locale)
+- [Tableau](#tableau)
+- [Table](#table)
+- [Table inline](#table-inline)
+- [Tableau de table](#tableau-de-table)
+- [Extension de nom de fichier](#extension-de-nom-de-fichier)
+- [Type MIME](#type-mime)
+- [Grammaire ABNF](#grammaire-abnf)
+
+
+Spécifications
+--------------
+
+* TOML est sensible à la casse.
+* Un fichier TOML doit être un document Unicode encodé UTF-8 valide.
+* Espace blanc signifie tabulation (0x09) ou espace (0x20).
+* Saut de ligne signifie LF (0x0A) ou CRLF (0x0D 0x0A).
+
+Commentaires
+------------
+
+Un symbole dièse marque le reste de la ligne en tant que commentaire, sauf à
+l'intérieur d'une chaîne de caractères.
+
+```toml
+# Ceci est un commentaire en ligne complète
+key = "value" # Ceci est un commentaire en fin de ligne
+other = "# Ceci n'est pas un commentaire"
+```
+
+Les caractères de contrôle autres que tabulation (U+0000 à U+0008, U+000A à
+U+001F, U+007F) ne sont pas autorisés dans les commentaires.
+
+Paire clé/valeur
+----------------
+
+Le bloc de construction principal d'un document TOML est la paire clé/valeur.
+
+Les clés sont à gauche du signe égal et les valeurs sont à droite. Les espaces
+blancs sont ignorés autour des noms et des valeurs de clé. La clé, le signe
+égal et la valeur doivent être sur la même ligne (bien que certaines valeurs
+puissent être réparties sur plusieurs lignes).
+
+```toml
+clé = "valeur"
```

Comment:
need to check if **é** must be write as Unicode value

Reply:
It can written like that :
cle = "valeur"

It's still understandable in, french